pom.xml

```diff
@@ -75,6 +75,13 @@
         </configuration>
       </plugin>
       <plugin>
+      	<groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+    		<argLine>-Xmx512m</argLine>
+  		</configuration>
```

Comment:
Yes, formatting wasn't correct. Next PR will fix it. @yogidevendra please check your editor settings.